### PR TITLE
Add gtk::Clipboard::request_uris()

### PR DESCRIPTION
**Pull request by YaLTeR**
Did I do this right?

### CODE REVIEW

File: src/clipboard.rs [hunk 1]
```diff
@@ -68,4 +68,27 @@ impl Clipboard {
...
+
+    pub fn request_uris<P: FnOnce(&Clipboard, &[glib::GString]) + 'static>(&self, callback: P) {
+        let callback_data: Box_<P> = Box_::new(callback);
+        unsafe extern "C" fn callback_func<P: FnOnce(&Clipboard, &[glib::GString]) + 'static>(
+            clipboard: *mut gtk_sys::GtkClipboard,
+            uris: *mut *mut c_char,
+            data: glib_sys::gpointer,
+        ) {
+            let clipboard = from_glib_borrow(clipboard);
+            let uris = from_glib_borrow(uris);
```
Comment (GuillaumeGomez) [inline src/clipboard.rs:80]: Just want to be sure here: does `glib_from_borrow` handle correctly pointers of pointers and stop at the first `0`/`NULL`?

Maybe you know @sdroege ?

**Comment by GuillaumeGomez** [comment]
Just something I'm wondering about but otherwise, it looks good to me, thanks!

File: src/clipboard.rs (hunk 1 above)
Comment (sdroege) [reply src/clipboard.rs:80]: I don't think this compiles

Comment (sdroege) [reply src/clipboard.rs:80]: See this for an example

```rust
    pub fn get_uris(&self) -> Vec<GString> {
        unsafe {
            FromGlibPtrContainer::from_glib_full(gtk_sys::gtk_selection_data_get_uris(
                self.to_glib_none().0,
            ))
        }
    }
```

Comment (YaLTeR) [reply src/clipboard.rs:80]: Should this be `from_glib_full`, `container` or `none`? The docs have no transfer annotation.

Comment (sdroege) [reply src/clipboard.rs:80]: `none`

Comment (YaLTeR) [reply src/clipboard.rs]: Okay, take two.

**Review by sdroege — APPROVED** [review]

**Comment by sdroege** [comment]
@GuillaumeGomez Good to be merged apart from travis timing out

**Review by GuillaumeGomez — APPROVED** [review]

**Comment by GuillaumeGomez** [comment]
Thanks!

**Pull request merged**